pkg/nfd-worker/nfd-worker.go

```diff
@@ -525,6 +548,14 @@ func (w *nfdWorker) configure(filepath string, overrides string) error {
 		s.SetConfig(c.Sources[s.Name()])
 	}
 
+	w.sourceEvent = make(chan *source.FeatureSource)
+	eventSources := source.GetAllEventSources()
+	for _, s := range eventSources {
```

Comment:
I think we should move these to `configureCore()`.

Also, we should probably honor the feature-sources flag/config so that if somebody does e.g. `-feature-sources=all,-local` it doesn't get activated.

Reply:
> Also, we should probably honor the feature-sources flag/config so that if somebody does e.g. `-feature-sources=all,-local` it doesn't get activated.

This is still unresolved, if I'm not mistaken